packages/pigment-css-react/tests/useTheme/fixtures/useTheme.input.js

```diff
@@ -0,0 +1,125 @@
+import { useTheme } from '../zero-styled';
+
+export const Fade = React.forwardRef(function Fade(props, ref) {
+  const theme = useTheme();
+  const defaultTimeout = {
+    enter: theme.transitions.duration.enteringScreen,
+    exit: theme.transitions.duration.leavingScreen,
+  };
+
+  const {
+    addEndListener,
+    appear = true,
+    children,
+    easing,
+    in: inProp,
+    onEnter,
+    onEntered,
+    onEntering,
+    onExit,
+    onExited,
+    onExiting,
+    style,
+    timeout = defaultTimeout,
+    // eslint-disable-next-line react/prop-types
+    TransitionComponent = Transition,
+    ...other
+  } = props;
+
```

Comment:
I think the test input should be simplified. A basic component that imports `useTheme()` should be enough.